shared/engine/transport-shared.js

```diff
@@ -39,6 +140,22 @@ class TransportShared extends RobustTransport {
       return new Error(JSON.stringify(err))
     }
   }
+
+  invoke (arg: Object, cb: any) {
+    // args needs to be wrapped as an array for some reason so lets just do that here
+    const wrappedArgs = {
+      ...arg,
+      args: [arg.args || {}],
+    }
+
+    const wrappedInvoke = _wrap((args) => {
```

Comment:
because _wrap does the logging so if you don't pass it in , it won't print it out